.github/workflows/release.yml

```
@@ -0,0 +1,254 @@
+# Robrix Release CI Workflow
+
+name: Robrix Release CI
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'    # Release Version Tags
+      - 'v*.*.*-*'  # Pre-release Version Tags
+  workflow_dispatch:
+    inputs:
+      target_platforms:
+        description: 'Target platforms for the release'
+        required: true
+        type: choice
+        options:
+          - 'All'                         # Build for all platforms
+          - 'linux-ubuntu-24.04'          # Build for Ubuntu 24.04 both x86_64 and aarch64
+          - 'linux-ubuntu-24.04-x86_64'   # Build for Ubuntu 24.04 x86_64
+          - 'linux-ubuntu-24.04-aarch64'  # Build for Ubuntu 24.04 aarch64
+          - 'linux-ubuntu-22.04'          # Build for Ubuntu 22.04 both x86_64 and aarch64
+          - 'linux-ubuntu-22.04-x86_64'   # Build for Ubuntu 22.04 x86_64
+          - 'linux-ubuntu-22.04-aarch64'  # Build for Ubuntu 22.04 aarch64
+          - 'macos-14-arm'                # Build for MacOS 14 (Apple Silicon)
```

nit: everything else uses aarch64, why is this "arm"?
```suggestion
          - 'macos-14-aarch64'            # Build for MacOS 14 (Apple Silicon)
```